src/modules/shortcut_guide/shortcut_guide.cpp

```diff
@@ -134,12 +134,7 @@ void OverlayWindow::enable()
         winkey_popup->set_theme(theme.value);
         target_state = std::make_unique<TargetState>(pressTime.value);
         winkey_popup->initialize();
-
-        hook_handle = SetWindowsHookEx(WH_KEYBOARD_LL, hookProc, GetModuleHandle(NULL), NULL);
-        if (!hook_handle)
-        {
-            throw std::runtime_error("Cannot install keyboard listener");
-        }
+        hook_handle = SetWindowsHookEx(WH_KEYBOARD_LL, lowLevelKeyboardProc, GetModuleHandle(NULL), NULL);

```

Comment:
This is significant error and in this case we want to provide some type of feedback, a message box or a telemetry event.

Reply:
Adding a message box.